jwst/jump/twopoint_difference.py

```diff
@@ -19,15 +19,23 @@
 
 HUGE_NUM = np.finfo(np.float32).max
 
-def find_crs(data, gdq, read_noise, rej_threshold, nframes):
+
+#def find_crs(indata, ingdq, read_noise, rej_threshold, nframes):
+def find_crs(indata):
 
     """
     Find CRs/Jumps in each integration within the input data array.
     The input data array is assumed to be in units of electrons, i.e. already
     multiplied by the gain. We also assume that the read noise is in units of
     electrons.
     """
-
+    if isinstance(indata, tuple):
+        read_noise = indata[2]
+        rej_threshold = indata[3]
+        nframes = indata[4]
+        data = indata[0]
+        ingdq = indata[1]
```

Comment:
Minor nit, but why not list these in the order in which they appear in the tuple (i.e. 0, 1, 2, 3, 4) for the sake of readability and following what's being passed where?